### ISSUE TITLE
The problem about EmbeddingBag module

### ISSUE THREAD

**Issue by santisy** [pytorch/pytorch#1864]
Code as below:
```python
import torch
from torch.autograd import Variable
import torch.nn as nn

a = Variable(torch.LongTensor(2, 2).fill_(1).cuda())
embedding_bag = nn.EmbeddingBag(2, 10)
embedding_bag(a)
```
raised an `AttributeError`

It seems https://github.com/pytorch/pytorch/blob/master/torch/nn/modules/sparse.py#L200 should be `offsets = input.data.new(input.size(0)).fill_(input.size(1))`?
I build from source with commit `67968cb`. python 2.7, cuda 8.0, cudnn 6

**Comment by LiyuanLucasLiu** [comment]
It seems that the EmbeddingBag has not been included in pytorch 0.1.12

**Comment by soumith** [comment]
thanks for reporting this. fixed via https://github.com/pytorch/pytorch/commit/20ce45b0c367ec860f84dce31bb40c6050fb9255
